cloud_build/deploy.yaml

```diff
@@ -1,20 +1,16 @@
 steps:
   - name: gcr.io/cloud-builders/git
     args: ['submodule', 'update', '--init', '--recursive']
-  - name: gcr.io/cloud-builders/docker
-    entrypoint: '/bin/bash'
-    args:
-      - '-c'
-      - |-
-        set -e
-        echo "Building the website using a makefile..."
-        make build
-        make write-prod-robots
   - name: gcr.io/flutter-dev-230821/firebase-ghcli
```

Comment:
Where did the step to build the static files go? Is that the equivalent of 'npm run prod' now?

Reply:
If so, could we make that "build_prod_site_files" or something that makes it clearer that we're building the files?

Reply:
Yep! `npm run build` builds the static site files for staging. I didn't need the earlier step as we don't use Docker now and the `gcr.io/flutter-dev-230821/firebase-ghcli` has Node available to use.

Reply:
> Could we make that "build_prod_site_files" or something ...

Could you specify what you're referring to? Do you mean the name of the `npm build` command?